examples/mysql-query-operation/mysql_query_operation.bal

```diff
@@ -18,77 +14,66 @@ type Customer record {|
 
 public function main() returns error? {
     // Runs the prerequisite setup for the example.
-    check beforeExample();
+    check initialize();
 
-    // Initializes the MySQL client.
+    // Initializes the MySQL client. The `mysqlClient` can be reused to access database throughout the application execution.
     mysql:Client mysqlClient = check new (user = "root",
-            password = "Test@123", database = "MYSQL_BBE");
+            password = "Test@123", database = "CUSTOMER");
 
-    // Select the rows in the database table via the query remote operation.
-    // The result is returned as a stream and the elements of the stream can
-    // be either a record or an error. The name and type of the attributes 
-    // within the record from the `resultStream` will be automatically 
-    // identified based on the column name and type of the query result.
+    float creditLimit = 5000;
+
+    // Query table with a condition.
     stream<Customer, error?> resultStream =
-            mysqlClient->query(`SELECT * FROM Customers`);
+            mysqlClient->query(`SELECT * FROM Customers
+                                WHERE creditLimit > ${creditLimit};`);
 
-    // If there is any error during the execution of the SQL query or
-    // iteration of the result stream, the result stream will terminate and
-    // return the error.
+    // Iterate the result stream.
```

Comment:
```suggestion
    // Iterates the result stream.
```

Reply:
Fixed in [4621ad1](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/4621ad1b1e00a60859c191de19de2041e0516f79)